NtupleTools/python/embedElectronIDs.py

```diff
@@ -47,6 +47,26 @@ def embedElectronIDs(process, use25ns, eSrc):
         cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring15NonTrig25nsV1Categories"),
         ]
 
+    # N-1 results
+    nMinusOneNames = ['GsfEleEffAreaPFIsoCut_0']
```

Comment:
@lgray Why is this the name of the cut? When I look here: https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_tools.py#L262 I see without the "_0".